src/site/xdoc/checks/misc/translation.xml.template

```diff
@@ -93,17 +93,29 @@ messages.properties: Key 'ok' missing.
         </p>
         <macro name="example">
           <param name="path"
-                 value="resources/com/puppycrawl/tools/checkstyle/checks/translation/Example1.txt"/>
+                 value="resources/com/puppycrawl/tools/checkstyle/checks/translation/Example1.java"/>
           <param name="type" value="config"/>
         </macro>
         <p id="Example1-code">
-          Example:
+          Example1:
```

Comment:
It took me a while to understand first example.....

please add one more block here to explain content of folder (attention no messages.translations ):
application.properties
messages.properties

please change property to `<property name="fileExtensions" value="properties"/>` 

this example will show how Check differentiate config files and transaltion files.